hpc_ch1_manuel.ipynb

```diff
@@ -4,7 +4,8 @@
    "cell_type": "markdown",
```

Comment:
<p>hier fehlt wohl ein Wort in diesem Satz.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rootpi2203/hs22-mc1/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
behoben